tests/virt_autotest/reset_partition.pm

```diff
@@ -0,0 +1,109 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+package reset_partition;
+# Summary: reset_partition: Ensure and reset hard disk partition for virt_autotest.
+# Maintainer: Leon Guo <xguo@suse.com>
+
+use base "virt_feature_test_base";
+use strict;
+use warnings;
+use testapi;
+use virt_utils;
+use ipmi_backend_utils;
+use Utils::Architectures;
+use utils qw(zypper_call);
+use version_utils 'is_sle';
+
+sub reset_partition {
+    my $self       = shift;
+    my $new_disk   = "";
+    my $virt_point = "/var/lib/libvirt/";
+
+    # Get some debug info about hard disk topology
+    record_info("Report file system disk space", script_output("df"));
+    record_info("Find mounted filesystem",       script_output("findmnt"));
+    record_info("Check with /var/lib/libvirt",   script_output("ls -al /var/lib/libvirt"));
+    script_run 'df --output=avail /var/lib/libvirt';
+
+    my ($DEPLOY_DISK_NAME, $AVAILABLE_DISK_SIZE) = $self->get_active_disk_and_available_space();
+    # Disk which /var/lib/libvirt resides on
+    record_info('Detect Deployed DISK NAME for /var/lib/libvirt:', $DEPLOY_DISK_NAME);
+    # Ensure available disk space under /var/lib/libvirt
+    record_info('Detect Available DISK SIZE under /var/lib/libvirt:', $AVAILABLE_DISK_SIZE . 'GiB');
+    # Define the expected disk space for /var/lib/libvirt both on KVM or XEN
+    my $expected_libvirt_size = (get_var("XEN")) ? '60' : '40';
```

Comment:
Please stop using get_var("XEN"), use is_xen_host instead. We tend to remove duplicate suite settings.

Reply:
sure, had used with` ( is_xen_host )` already. 
Thanks for the reminder.